data_prep/prep_cms_data.py

```diff
@@ -29,41 +29,66 @@ def extract_star_ratings(df):
                 base_df[q] = df_ratings.values
 
         return base_df.reset_index(drop=True)
+def extract_questions_as_columns(df, care_type):
```

Comment:
These changes are unrelated to the marker color but tagged along accidentally. There is no effect for the user so i will leave these in.